### ISSUE TITLE
Error using lang.yaml in theme

### ISSUE THREAD

**Issue by over12** [rainlab/translate-plugin#567]
Hi guys!

I followed the tips in the document which explains how to put translations into an october theme.
For you to understand how I structured my code, follow below ...

Folder config with files lang.yaml
https://prnt.sc/rtci4k

My file lang-pt-br.yaml open
```
footer.development: 'Development with'
footer.by: 'By'
```
My file theme.yaml open
```
name: Theme Default
description: ''
author: 'Over12'
homepage: 'https://default.nt'
code: ''
translate: 
pt-br: config/lang-pt-br.yaml
```
In my HTML I inserted this code...
`{ development|_ }} <i class="fas fa-heart heart-icon text-danger"></i> {{ footer.by|_ }}`

But in the end it is not translated, the final result is empty the place that should show the message.
https://prnt.sc/rtcl2m

Just to remember in my configuration files I have already set the default language "pt-br" active.

Has anyone ever experienced this?

**Comment by mjauvin** [comment]
I see three problems:

1. looks like you're missing an indentation in theme.yaml in front of  "pt-br:"

2. you need to "quote" your translation strings like `{{ "footer.by" | _}}`

3. in your example you define "footer.development" but reference it without the footer prefix... should be `{{ "footer.development" | _ }}`

**Comment by over12** [comment]
Hi @mjauvin, thanks for answers.. 

The items you mentioned above is that I had done several tests in different ways, and when I made the post I forgot to go back to the original form that was in the guide.

Anyway, I did the issue of indentations and even then it doesn't work, the only thing I didn't try was to wrap the footer.development parameter in quotes, but it didn't work and it was like this in the Front.
https://prnt.sc/rtz8ht

Updated my Theme.yaml
Test 1:
```
translate: 
    "en": config/lang-en.yaml
    "pt-br": 'config/lang-pt-br.yaml'
```
Test 2:
```
translate: 
    en: config/lang-en.yaml
    pt-br: 'config/lang-pt-br.yaml'
```
I did new tests on theme.yml adding quotes between languages, between the path of the directory, but without success.
